### DISCUSSION TITLE
请教下各位，最近项目要上apollo，用哪个版本号呢，1.8.0？

### DISCUSSION THREAD

**Discussion by howHzz** [apolloconfig/apollo#3596] [Q&A] [answered]
- [ ] 我已经检查过[discussions](https://github.com/ctripcorp/apollo/discussions)
- [ ] 我已经搜索过[issues](https://github.com/ctripcorp/apollo/issues)
- [ ] 我已经仔细检查过[FAQ](https://ctripcorp.github.io/apollo/#/zh/faq/common-issues-in-deployment-and-development-phase)

**描述bug**

简洁明了地描述一下bug

**复现**

通过如下步骤可以复现：

1. 
2. 
3. 
4. 

**期望**

简介明了地描述你希望正常情况下应该发生什么

**截图**

如果可以，附上截图来描述你的问题

### 额外的细节和日志

- 版本：
- 错误日志
- 配置：
- 平台和操作系统

**Comment by Anilople** [comment, marked as answer]
参考 https://github.com/ctripcorp/apollo/releases

直接使用最新版即可
